lib/blade/sauce_labs_plugin/tunnel.rb

```diff
@@ -36,6 +36,7 @@ def create_child_process
       ChildProcess.build(tunnel_command, *tunnel_args).tap do |process|
         process.leader = true
         process.io.inherit! if debug?
+        process.environment.merge! tunnel_env
```

Comment:
Inject auth credentials into child process' environment. It inherits existing env vars, but this ensures that username/access_key in .blade.yml will be passed as env vars also.